renderer/locales/en.json

```diff
@@ -1,5 +1,210 @@
 {
-  "settings": {
-    "language": "Change Language"
+  "TITLE": "Upscayl",
+  "INTRO": "Introducing Upscayl Cloud!",
+  "HEADER": {
+    "GITHUB_BUTTON_TITLE": "Star us on GitHub 😁",
+    "DESCRIPTION": "AI Image Upscaler"
+  },
+  "FOOTER": {
+    "NEWS_TITLE": "UPSCAYL NEWS",
+    "COPYRIGHT": "Copyright ©",
+    "TITLE": "By ",
+    "LINK_TITLE": "The Upscayl Team"
+  },
+  "SETTINGS": {
+    "TITLE": "SETTINGS",
+    "CHANGE_LANGUAGE": { "TITLE": "Change Language" },
+    "IMAGE_COMPRESSION": {
+      "TITLE": "Image Compression",
+      "DESCRIPTION": "PNG compression is lossless, so it might not reduce the file size significantly and higher compression values might affect the performance. JPG and WebP compression is lossy."
+    },
+    "CUSTOM_MODELS": {
+      "TITLE": "ADD CUSTOM MODELS",
+      "BUTTON_FOLDER": "Select Folder",
+      "DESCRIPTION": "You can add your own models easily. For more details:",
+      "LINK_TITLE": "Custom Models Repository"
+    },
+    "CUSTOM_INPUT_RESOLUTION": {
+      "TITLE": "CUSTOM OUTPUT WIDTH",
+      "RESTART": "REQUIRES RESTART",
+      "DESCRIPTION": "Use a custom width for the output images. The height will be adjusted automatically. Enabling this will override the scale setting."
+    },
+    "DONATE": {
+      "DESCRIPTION": "If you like what we do :)",
+      "BUTTON_TITLE": "💎 DONATE"
+    },
+    "GPU_ID_INPUT": {
+      "TITLE": "GPU ID",
+      "DESCRIPTION": "Please read the Upscayl Documentation for more information.",
+      "ADDITIONAL_DESCRIPTION": "Enable performance mode on Windows for better results."
+    },
+    "IMAGE_FORMAT": {
+      "TITLE": "SAVE IMAGE AS",
+      "PNG": "PNG",
+      "JPG": "JPG",
+      "WEBP": "WEBP"
+    },
+    "IMAGE_SCALE": {
+      "TITLE": "Image Scale",
+      "IMAGE_SCALED_BY": "IMAGE SCALE ({scale}X)",
```

Comment:
![Slider has title "Image Scale" and description in smallcaps to the right "image scale (4x)"](https://github.com/user-attachments/assets/199d65c2-785e-4492-a986-dae3aec58b3f)
why is this like this lol

Reply:
Oh damn.. that's an oopsie, I'll check this